ui/src/components/ToastyBoi.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from "react";
+import Snackbar from '@material-ui/core/Snackbar';
+import { makeStyles } from '@material-ui/core/styles';
+import Slide from '@material-ui/core/Slide';
+
+import { colors } from "../styles/colors";
+import Autocomplete from "@material-ui/lab/Autocomplete";
+
+const { forwardRef, useImperativeHandle } = React;
+
+const ToastyBoi = forwardRef((props, ref) => {
+  const [state, setState] = React.useState<{ open?: boolean; message?: string}>({
```

Comment:
nit: more descriptive state name
```suggestion
  const [toastState, setToastState] = React.useState<{ open?: boolean; message?: string}>({
```